### DISCUSSION TITLE
Email could not be sent

### DISCUSSION THREAD

**Discussion by kamadam** [snipe/snipe-it#13404] [General]
Hello, 

I am fresh for this snapit inventory. I got error 
<img width="548" alt="image" src="https://github.com/snipe/snipe-it/assets/21234429/ef4718c8-c3a9-4dbd-8c1a-44ca78baaee6">

Below I have mentioned outgoing mail server entry
MAIL_DRIVER=smtp
MAIL_HOST=smtp.gmail.com
MAIL_PORT=587
MAIL_USERNAME=xxxxxxxxx@gmail.com 
MAIL_PASSWORD=
MAIL_ENCRYPTION=tls
MAIL_FROM_ADDR=xxxxxxxxxxx@gmail.com
MAIL_FROM_NAME='- IT Support'
MAIL_REPLYTO_ADDR=xxxxxxxxxx@gmail.com
MAIL_REPLYTO_NAME='Re: MLG - IT Support'
MAIL_AUTO_EMBED_METHOD='attachment'

please assist me, how to resolve it. above I have mentioned email and also i tried with outlook email account. I got same error.

**Comment by snipe** [comment]
We'd need the contents of the app logs. https://snipe-it.readme.io/docs/getting-help#step-3-check-your-app-and-server-logs
